src/main/java/com/aerospike/connect/outbound/format/BatchFormatter.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *
+ *  Copyright 2012-2022 Aerospike, Inc.
+ *
+ *  Portions may be licensed to Aerospike, Inc. under one or more contributor
+ *  license agreements WHICH ARE COMPATIBLE WITH THE APACHE LICENSE, VERSION 2.0.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+package com.aerospike.connect.outbound.format;
+
+import com.aerospike.connect.outbound.esp.EspOutboundMetadata;
+import com.aerospike.connect.outbound.pulsar.PulsarOutboundMetadata;
+import lombok.NonNull;
+
+import java.util.List;
+
+/**
+ * Format the Aerospike records into an outbound records for the outbound
+ * destination.
+ *
+ * <p>
+ * The implementing code should specify the type {@code T} applicable to the
+ * outbound destination - like {@link EspOutboundMetadata} for ESP (Event Stream
+ * Processing) destination, {@link PulsarOutboundMetadata} for Pulsar
+ * destination, etc.
+ *
+ * @param <T> the metadata associated with the outbound records.
+ */
+public interface BatchFormatter<T extends OutboundMetadata> {
+    /**
+     * Format records into a custom format.
+     *
+     * <p>
+     * This interface is identical to the {@link Formatter} interface. Here we
```

Comment:
Do not use "we", "us", etc. Here it should probably read "collection of BatchItems are passed as a parameter to the formatter" or something along those lines.